mne/preprocessing/xdawn.py

```diff
@@ -244,12 +244,14 @@ def fit(self, epochs, y=None):
         if self.correct_overlap == 'auto':
             self.correct_overlap = _check_overlapp(epochs)
 
+        epochs_data = X.reshape(X.shape[0], info['nchan'], X.shape[1] / 
+                                            info['nchan'])
         # Extract signal covariance
         if self.signal_cov is None:
             if self.correct_overlap:
                 sig_data = _construct_signal_from_epochs(epochs)
```

Comment:
When `overlap` is true, the functions above(to calculate `sig_data`) use `epochs.events[:, 0]`. I am quite confused on what to do here.